apps/freelance/src/pages/ProfileQuestions1/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { ProgressBar } from '@freelance/components';
+import { FormWrapper, Wrapper } from './styles';
+import { Form, Input, Select } from 'antd';
+import { DefInput } from '@freelance/components';
+
+const ProfileQuestions1 = () => {
+  const { t } = useTranslation();
+  const progressBarPer = 20;
```

Comment:
you constant will be initialising with each rerender, very very bad approach. all constants should be above the file
but this component is wrong in general this 20 value should come from props, in other case functionality of the component will be too limited

Reply:
fixed. Data comes from constant folder for now (Dmytro (PM) said that progress bar is not a focus of this sprint, so better logic will be created when asked)
![image](https://user-images.githubusercontent.com/95277626/200793873-9fbbf774-041e-45e6-8d45-0a802f6bfbdb.png)
![image](https://user-images.githubusercontent.com/95277626/200794052-611fb964-0667-45af-bd55-1bf054e6babf.png)